pkgs/top-level/all-packages.nix

```diff
@@ -22360,6 +22360,8 @@ with pkgs;
 
   graph-cli = callPackage ../tools/graphics/graph-cli { };
 
+  guacamole = callPackage ../servers/guacamole { };
```

Comment:
It seems like there are some issues with openssl 3 so I changed this to
`guacamole = callPackage ../servers/guacamole { openssl = pkgs.openssl_1_1; };`
... to get it working